requirements.txt

```diff
@@ -1,6 +1,17 @@
+certifi==2017.11.5
+chardet==3.0.4
 click==6.7
 Flask==0.12.2
+idna==2.6
 itsdangerous==0.24
 Jinja2==2.10
 MarkupSafe==1.0
-Werkzeug==0.13
\ No newline at end of file
+numpy==1.13.3
+pandas==0.21.1
+python-dateutil==2.6.1
+pytz==2017.3
+requests==2.18.4
+requests-mock==1.4.0
+six==1.11.0
+urllib3==1.22
+Werkzeug==0.13
```

Comment:
What do you all think about using Pipfile instead of requirements.txt? https://github.com/pypa/pipfile

I appreciate that all dependencies are pinned in this requirements.txt, but it is nice to keep the "first order" dependencies separate from the full list. Pipfile has a notion of a lock file, similar to how npm now works with package-lock.json.

Reply:
I'm leery of using it until it becomes stable.